app/containers/CreateUser/formUtils.js

```diff
@@ -0,0 +1,37 @@
+const isValidElement = elem => elem.name && elem.value
+
+// Only take into account text-like inputs and checked radios/checkboxes
+const isValidValue = elem => (
+  !['checkbox', 'radio'].includes(elem.type) || elem.checked
+)
+
+/**
+ * Utility function for converting form data to an object.
+ *
+ * @param {HTMLFormControlsCollection} formElements
+ */
+function formToObject(formElements) {
+  // Use reduce.call since formElements is not actually an array
+  const formData = [].reduce.call(formElements, (data, elem) => {
+    if (isValidElement(elem) && isValidValue(elem)) {
+      let value
+      if (elem.type == 'checkbox') {
```

Comment:
Should be `===`?

Reply:
Ahh, you're correct, surprised we didn't have an ESLint rule for this!

Reply:
If we don't we should probably add it!